flake8 is failing

At the moment the flake8 CI throws an error during setting up the environment.
```
##[error]Version 3.7.8 with arch x64 not found

Available versions:
2.7.18 (x64)
3.5.9 (x64)
3.6.12 (x64)
3.7.9 (x64)
3.8.5 (x64)
```

Potential fix to change https://github.com/wellcometrust/WellcomeML/blob/11f0ee28902831f69effd8497ab689055045f804/.github/workflows/main.yml#L22 to 3.7.9

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -19,7 +19,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
-          python-version: 3.7.8
+          python-version: 3.7.9
           architecture: x64
       - name: Checkout Master
         uses: actions/checkout@master

